<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>199</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000000</width>
    <height>1000000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройка моделирования</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Введите данные для моделирования</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="Label">
          <property name="text">
           <string>Кол-во видов товаров:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="ProductsSpinBox">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>17</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="Label_2">
          <property name="text">
           <string>Торг. точки:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="ShopsSpinBox">
          <property name="minimum">
           <number>3</number>
          </property>
          <property name="maximum">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Label_3">
          <property name="text">
           <string>Период моделирования:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="DaysSpinBox">
          <property name="minimum">
           <number>7</number>
          </property>
          <property name="maximum">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QDialogButtonBox" name="OkButton">
        <property name="standardButtons">
         <set>QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
